packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -18,6 +18,9 @@
                 padding-top: 0.4rem;
             }
         }
+        .dc-mobile-full-page-modal {
+            opacity: 1 !important;
```

Comment:
Had to set the opacity to 1 and use !important, when the user goes into p2p page from outside of p2p, and then goes to my ads page and then clicks the Add button, the modal would not show, but it is rendered onto the dom with an opacity of 0.

Reply:
can we use `is_modal_open` prop instead?